static/src/javascripts/bootstraps/enhanced/media/main.js

```diff
@@ -326,7 +326,9 @@ define([
                         resolve();
                     }
 
-                    if (abVideoAutoplay.variant === 'autoplay') {
+                    var contentType = config.page.contentType;
+                    var isArticleOrLiveBlog = contentType === 'Article' || contentType === 'LiveBlog';
+                    if (abVideoAutoplay.variant === 'autoplay' && isArticleOrLiveBlog) {
```

Comment:
Brilliant - it would be good to find a way to force someone into a test though, even while using the `isInVariant`.